Source/Shaders/BillboardCollectionFS.glsl

```diff
@@ -55,4 +64,39 @@ void main()
 #endif
 
     czm_writeLogDepth();
+
+#ifdef CLAMP_TO_GROUND
+    if (v_eyeDepth > -2000.0) {
+        vec2 adjustedST = v_textureCoordinates - v_textureOffset.xy;
+        adjustedST = adjustedST / (v_textureOffset.z - v_textureOffset.x, v_textureOffset.w - v_textureOffset.y);
```

Comment:
I'm surprised this compiled. Add use the `vec2` constructor for the denominator.

Reply:
Thanks haha.  I hardly ever glsl so I'm not very familiar with the syntax